scripts/toltec/templates/listing.html

```diff
@@ -54,7 +54,7 @@
         <h1>Toltec Package Listing</h1>
 
         {% for section, packages in sections.items() %}
-            <h2>{{ section }}</h2>
+            <h2 id="{{ section }}">{{ section }}</h2>
```

Comment:
```suggestion
            <h2 id="section-{{ section }}">{{ section }}</h2>
```
If we add anchors for specific packages in the future, I'd like to avoid to change links to avoid conflict.